easybuild/easyblocks/b/binutils.py

```diff
@@ -86,13 +94,14 @@ def configure_step(self):
             # determine list of 'lib' directories to use rpath for;
             # this should 'harden' the resulting binutils to bootstrap GCC
             # (no trouble when other libstdc++ is build etc)
+            lib_paths = self.determine_used_library_paths()
 
             # The installed lib dir must come first though to avoid taking system libs over installed ones, see:
-            # https://github.com/easybuilders/easybuild-easyconfigs/issues/10056
-            # Escaping: Double $$ for Make to get literal $ORIGIN in the file
-            libdirs = [r'$$ORIGIN/../lib'] + self.determine_used_library_paths()
+            # https://github.com/easybuilders/easybuild-easyconfigs/issues/10056;
+            # double $$ to get literal $ORIGIN in the file when command is run
```

Comment:
This comment is actually wrong. The $$ is (still) for make as in the comment above.

Reply:
The comment (which was there originally) didn't make sense to me, so I tweaked it.

Which `make` is being referred to here?

Please open an issue or PR to follow up, keeping track of things in closed/merged PRs is close to impossible to keep up with...

Reply:
https://github.com/easybuilders/easybuild-easyblocks/pull/2327